src/plugins/vis_builder/public/plugin.ts

```diff
@@ -154,7 +216,7 @@ export class VisBuilderPlugin
 
   public start(
     core: CoreStart,
-    { data, expressions }: VisBuilderPluginStartDependencies
+    { expressions, data }: VisBuilderPluginStartDependencies
```

Comment:
nit - I'd expect that the old version is preferable, with the properties alphabetized...